Lombiq.HelpfulLibraries.OrchardCore/TagHelpers/EditorFieldSetTagHelper.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Localization;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using System;
+using System.Threading.Tasks;
+
+namespace Lombiq.HelpfulLibraries.OrchardCore.TagHelpers;
+
+[HtmlTargetElement("fieldset", Attributes = "asp-for")]
+public class EditorFieldSetTagHelper : TagHelper
```

Comment:
I wanted to suggest that it should rather be named:
```suggestion
public class EditorFieldsetTagHelper : TagHelper
```
... but even [the official specification](https://html.spec.whatwg.org/multipage/form-elements.html#htmlfieldsetelement) uses `FieldSet`, so you were right on point!

Reply:
As far as I know `fieldset` is not a real word; it's a field set or a set of fields. HTML does the same with `textarea` too - it would be cleaner as `field-set` and `text-area`, but early Internet was all about saving those extra bytes of storage and bandwidth.

Reply:
I agree.